thrift-bindings/generate.sh

```diff
@@ -3,6 +3,7 @@
 GEN_LANG="$1"
 
 THRIFT_FILE="remote_controller.thrift"
+THRIFT=${THRIFT:-thrift}
```

Comment:
Is the name of the binary changed or you did this because the `thrift` binary is not in your path?

Reply:
nope name of the binary is the same as far as I see.